Src/BootCamp.Chapter/Credentials.cs

```diff
@@ -1,13 +1,20 @@
-﻿namespace BootCamp.Chapter
+﻿using System;
+using System.Data;
+
+namespace BootCamp.Chapter
 {
     // TODO: make a struct and add validation and other needed methods (if needed)
-    public class Credentials
+    public readonly struct Credentials : IEquatable<Credentials>
```

Comment:
Implementing IEquatable was suggested by MihailFox so no props to me. I did my homework though and researched it fairly well so I can understand and reimplement this again in the future.

Reply:
IEquatable is needed for comparing. There is indeed a use case here. So that is fine.